frontendUiApi/build.gradle

```diff
@@ -0,0 +1,8 @@
+vavr()
```

Comment:
This module is basically for the stuff that `frontendUiTableImpl` and `frontendUiRootImpl` need to pass b/w each other (dropdown, table manager)